client/blocks/reader-related-card-v2/index.jsx

```diff
@@ -95,6 +98,9 @@ export function RelatedPostCard( { post, site, onPostClick = noop, onSiteClick =
 		'has-thumbnail': !! featuredImage
 	} );
 
+	// TODO: is this okay?
+	const style = { WebkitLineClamp: lineClamp };
```

Comment:
here's where we do it in the in stream recs pr https://github.com/Automattic/wp-calypso/pull/9490/files#diff-5fc366969c3080c52fb50121347ef0e3R469

maybe we move that bit up the chain a bit and into the related card block?